ansible/templates/whisk.properties.j2

```diff
@@ -80,8 +80,8 @@ db.port={{ db_port }}
 db.username={{ db_username }}
 db.password={{ db_password }}
 db.prefix={{ db_prefix }}
-db.whisk.actions={{ db_prefix }}whisks
-db.whisk.auths={{ db_auth }}
+db.whisk.actions={{ db.whisk.actions }}
```

Comment:
I'm seeing that this property keeps changing.  See https://github.com/openwhisk/openwhisk/issues/197 .

Since it has a potential breaking impact in the ENV VARs, that would have to change too, it would be good to avoid changing it unless there's critical functionality that depends on it.